.github/workflows/multi-approvers.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2025 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'multi-approvers'
+
+on:
+  pull_request:
+    types:
+    - 'opened'
+    - 'edited'
+    - 'reopened'
+    - 'synchronize'
+    - 'ready_for_review'
+    - 'review_requested'
+    - 'review_request_removed'
+  pull_request_review:
+    types:
+    - 'submitted'
+    - 'dismissed'
+
+permissions:
+  actions: 'write'
+  contents: 'read'
+  pull-requests: 'read'
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  multi-approvers:
+    uses: 'abcxyz/pkg/.github/workflows/multi-approvers.yml@main'
```

Comment:
This looks scary, `abcxyz` is some private org. Why are we taking dependencies on this?